lib/core/auth.js

```diff
@@ -22,8 +22,8 @@ module.exports = function (config) {
             return token || localStorage.KW_TOKEN;
         },
 
-        login: function () {
-            location.href = config.WORLD_URL + '/external/login?redirect_url=' + location.href;
+        login: function (token) {
```

Comment:
Does this mean that now you already have the token when you call the login function?